src/lib/util.js

```diff
@@ -2,7 +2,7 @@ export function isValidRepoUrl(url) {
   url = url.trim();
```

Comment:
i think if you change this to `url = url.toLowerCase().trim();`, the tests will pass (they pass for me locally).

Reply:
Done :thumbsup:
:crossed_fingers:(finger crossed)